libs_js/next_gen_ui_react/README.md

```diff
@@ -1,61 +1,64 @@
-# Next Gen UI Agent Dynamic UI NPM Package
+# Next Gen UI - Patternfly React Renderer
```

Comment:
I will review this later, after we discuss whole concept of READMEs at team meeting